tests/test_utils_python.py

```diff
@@ -238,13 +237,11 @@ def __call__(self, a, b, c):
         self.assertEqual(get_func_args(str.split, stripself=True), ["sep", "maxsplit"])
         self.assertEqual(get_func_args(" ".join, stripself=True), ["iterable"])
 
-        if platform.python_implementation() == "CPython":
-            # doesn't work on CPython: https://bugs.python.org/issue42785
-            self.assertEqual(get_func_args(operator.itemgetter(2)), [])
-        elif platform.python_implementation() == "PyPy":
-            self.assertEqual(
-                get_func_args(operator.itemgetter(2), stripself=True), ["obj"]
-            )
+        # Didn't used to work on CPython: https://bugs.python.org/issue42785
+        self.assertIn(
+            get_func_args(operator.itemgetter(2), stripself=True),
+            [[], ["obj"], ["args", "kwargs"]],
```

Comment:
Is `["args", "kwargs"]` what CPython returns now? If so, it might be best to keep the old if/else for PyPy, and assert CPython for `[[], ["args", "kwargs"]]` instead.

Reply:
Alright, I'll update it.